microsoft/testsuites/gpu/gpusuite.py

```diff
@@ -245,13 +251,29 @@ def _ensure_driver_installed(
     if gpu_feature.is_module_loaded():
         return
 
-    gpu_feature.install_compute_sdk()
+    ##gpu_feature.install_compute_sdk()
+    ##gpu_feature.install_driver_script()
+
+    '''node.execute(
+        ##f"./microsoft/runbook/azure/install_dependencies_alma.sh alma8.6",
+        ##f"touch /tmp/test29",
+        f"./test.sh",
+        shell=True,
+        sudo=True,
+    )'''
+    ##node.execute("./test.sh, cwd=./microsoft/testsuites/gpu, shell=True, sudo=True")
+
+    test_script = CustomScriptBuilder(Path(__file__).parent.joinpath("."), ["install_dependencies_alma.sh"])
```

Comment:
This should be seperated into multiple tools. The scripts are hard to maintain. Please let us know if you are finding it difficult to implement any tools. Further, you can use `node.execute` to run a command directly

Reply:
I was told by Chi that I should use CustomScriptBuilder to upload scripts to remove node to run. `node.execute` will only run local scripts. In this case it does not work.

Reply:
I left the `node.execute` block (commented out) just for reminder. Please let me know if you want me to remove it.

Reply:
Can you please remove these comments before we merge the PR? We want to make sure that only clean changes are going to the main branch